_posts/2019-08-22-bisq-dao-first-four-cycles.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "Bisq DAO: The First 4 Cycles"
+author: Steve Jain
+excerpt: "The Bisq DAO launched 4 months ago, after more than 4 years of development. It has now completed 4 voting cycles. In this post, we provide an update on how it's worked out so far. <br><br>"
+---
+
+<div class='responsive-youtube-container'>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/zjdB5_r3mG8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
I see many other links in the website use `youtube-nocookie.com`. I cannot find any proper documentation about this service, but seems to filter most of the cookies coming from youtube, so i think it's better to use it.